cmd/lifecycle/analyzer.go

```diff
@@ -130,7 +131,7 @@ func (aa analyzeArgs) analyze(group lifecycle.BuildpackGroup, cacheStore lifecyc
 	}
 
 	analyzedMD, err := (&lifecycle.Analyzer{
-		Buildpacks: group.Group,
+		Buildpacks: append(privGroup.Group, group.Group...),
```

Comment:
What happens with cached launch layers, it seems like we are only analyzing/restoring layers created on the build image?

Reply:
This first draft doesn't have the `extend` phase at all, so the layers being restored here for stackpacks are JUST the ones that ran on build. Does that make sense? I might be misunderstanding the question though.